### PR TITLE
feat: Add full beaconchain explorer

### PR DESCRIPTION
**Pull request by peterbitfly**
This PR adds the ability to provision the full version of the beaconcha.in explorer instead of the light version currently in use. While heavier on the resource usage, the full version also provides insight into the EL data of the test network (accounts, tokens, transactions).

Which explorer is started can be defined using the new `explorer_version` config field. Use `light` to launch the light version and `full` to launch the full explorer.

Note: The full explorer relies on Google bigtable for CL & EL data storage. Instead of the original bigtable environment the explorer uses a persistent emulator based on SQLite (https://github.com/gobitfly/little_bigtable) that has been tuned to this specific use case. The emulator should support testnets up to a few thousand validators.

Furthermore the integration currently relies on a branch of the explorer that has not yet been merged to master. Once that is completed we will submit a subsequent PR that updates the used image.

### CODE REVIEW

**Comment by piwonskp** [comment]
Hey @peterbitfly, very useful PR! However relays subpage is empty. Do you know what may be causing this?

**Comment by h4ck3rk3y** [comment]
@peterbitfly Thank you for the PR! This is beautiful work. Sorry we missed this. A couple of comments 

Can we change the postgres / redis services to use https://github.com/kurtosis-tech/postgres-package & https://github.com/kurtosis-tech/redis-package?

I see there are merge conflicts since this was created, do you have time to fix the merge conflicts?

File: README.md [hunk 1]
```diff
@@ -181,6 +181,12 @@ To configure the package behaviour, you can modify your `network_params.json` fi
     //  - params for the CL nodes will be ignored (e.g. CL node image, CL node extra params)
     "launch_additional_services": true,
 
+    // Which blockchain explorer should be used
```
Comment (h4ck3rk3y) [inline README.md:184]: Thank you for being so thoroguh!

File: src/full_beaconchain/full_beaconchain_launcher.star [hunk 2]
```diff
@@ -0,0 +1,255 @@
...
+    plan,
+        config_template,
+        cl_client_contexts,
+        el_client_contexts,
+    ):
+
+    db_services = plan.add_services(
+        configs={
+	        # Add a Postgres server
+            "explorer-postgres": ServiceConfig(
```
Comment (h4ck3rk3y) [inline src/full_beaconchain/full_beaconchain_launcher.star:36]: can we use the https://github.com/kurtosis-tech/redis-package and https://github.com/kurtosis-tech/postgres-package

for this?

File: src/package_io/parse_input.star [hunk 3]
```diff
@@ -156,6 +156,7 @@ def parse_input(plan, input_args):
 			mev_flood_seconds_per_bundle = result["mev_params"]["mev_flood_seconds_per_bundle"]
 		),
 		launch_additional_services=result["launch_additional_services"],
+		explorer_version=result["explorer_version"],
```
Comment (h4ck3rk3y) [inline src/package_io/parse_input.star:159]: this file has changed heavily, since we merged parsing; but these two changes make sense

**Review by h4ck3rk3y — APPROVED** [review]

**Comment by h4ck3rk3y** [comment]
Closing this in favor of #253  - thank you for your contribution! This means a lot :)

Since eth2/eth-network-package have been merged - adding features / fixing bugs should be a lot easier now

**Pull request closed without merging**